.cirrus.yml

```diff
@@ -72,8 +72,22 @@ low_scale_task:
   test_script:
     - 'sed -i "s/^  log_cmds\: False/  log_cmds\: True/"
         test-scenarios/ovn-low-scale*.yml'
-    - ./do.sh run test-scenarios/ovn-low-scale.yml low-scale
-    - ./do.sh run test-scenarios/ovn-low-scale-ic.yml low-scale-ic
+    - 'for ip4 in True False; do
+        for ip6 in True False; do
```

Comment:
Can we create a matrix instead of putting 3 different tests into one?

Reply:
It took some effort and it has some downsides but I pushed it to the PR as it is cleaner.  One thing we might have to adjust is the matrix size.  Especially with the thought in mind that we'll be adding OpenStack scenarios too.